kaldb/src/main/proto/metadata.proto

```diff
@@ -64,3 +64,37 @@ message SearchMetadata {
   // url
   string url = 3;
 }
+
+message RecoveryNodeMetadata {
+  enum RecoveryNodeState {
+    FREE = 0;
+    ASSIGNED = 1;
+    RECOVERING = 2;
+  }
+
+  // Name of recovery node
+  string name = 1;
+
+  // Name of the recovery task
+  string recoveryTaskName = 2;
+
+  // State of the recovery node
+  RecoveryNodeState recoveryNodeState = 3;
+
+  // Last updated timestamp
+  int64 updatedTimeUtc = 4;
```

Comment:
`lastUpdatedTimeUtc` would be a better name. Same applies to CacheNodeMetadata also.

Reply:
Leaving this for now per our Slack discussion, and we'll refactor all these time variables at once to a consistent pattern.